session.go

```diff
@@ -677,6 +689,10 @@ func (s *session) handleRetryPacket(p *receivedPacket, hdr *wire.Header) bool /*
 
 func (s *session) handleUnpackedPacket(packet *unpackedPacket, rcvTime time.Time) error {
 	if len(packet.data) == 0 {
+		// Ignore empty packets if mitigation is turned on.
+		if s.config.AttackTimeout > 0 && packet.encryptionLevel != protocol.Encryption1RTT {
+			return qerr.Error(qerr.NoError, "empty packet")
```

Comment:
Something feels strange about using `NoError` to describe these recoverable error situations, since they are in fact protocol errors.  I think you might be better off defining a new error type like `qerr.Recoverable`*, or perhaps adding an attribute like `QuicError.Ignorable`.

*Perhaps using the new [`Unwrap` pattern](https://blog.golang.org/experiment).